lib/src/ext/KHR_materials_dispersion/khr_materials_dispersion.dart

```diff
@@ -0,0 +1,63 @@
+// Copyright 2024 The Khronos Group Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+library gltf.extensions.khr_materials_dispersion;
+
+import 'package:gltf/src/base/gltf_property.dart';
+import 'package:gltf/src/ext/extensions.dart';
+
+const String KHR_MATERIALS_DISPERSION = 'KHR_materials_dispersion';
+
+const String DISPERSION = 'dispersion';
+
+const List<String> KHR_MATERIALS_DISPERSION_MEMBERS = <String>[
+  DISPERSION,
+];
+
+class KhrMaterialsDispersion extends GltfProperty {
+  final double dispersion;
+
+  KhrMaterialsDispersion._(
+      this.dispersion, Map<String, Object> extensions, Object extras)
+      : super(extensions, extras);
+
+  static KhrMaterialsDispersion fromMap(
+      Map<String, Object> map, Context context) {
+    if (context.validate) {
+      checkMembers(map, KHR_MATERIALS_DISPERSION_MEMBERS, context);
+    }
+
+    final dispersion =
+        getFloat(map, DISPERSION, context, min: 0, def: 0, standalone: 0);
```

Comment:
Please remove `standalone`; it's for values outside of a continuous range, e.g., `ior == 0 || ior > 1`.